cmd/kubeadm/app/cmd/certs.go

```diff
@@ -97,7 +97,7 @@ func newCmdCertsUtility(out io.Writer) *cobra.Command {
 		Short:   "Commands related to handling kubernetes certificates",
 	}
 
-	cmd.AddCommand(newCmdCertsRenewal(out))
```

Comment:
i think we can omit the output change. it's out of scope for this PR, but also i think we should wire it to print to the provided io.Writer eventually.

Reply:
make sense, will revert it back.